resources/quality/goofoo/pva/goofoo_open_0.40_pva_efine.inst.cfg

```diff
@@ -0,0 +1,25 @@
+[general]
+version = 4
+name = Extra Fine Quality
+definition = goofoo_open
+
+[metadata]
+setting_version = 17
+type = quality
+quality_type = efine
+material = goofoo_pva
+variant = 0.4mm Nozzle
+
+[values]
+wall_thickness = =line_width*2
+adhesion_type = none
+support_pattern = everywhere
+support_pattern = grid
```

Comment:
Support pattern is mentioned twice. Please remove one of the two.

Reply:
Thank you for your suggestions! I have modified it! Please check it! Thanks!